packages/wrangler/src/init.ts

```diff
@@ -591,10 +594,12 @@ export async function initHandler(args: InitArgs) {
 					`/accounts/${accountId}/workers/services/${fromDashScriptName}/environments/${defaultEnvironment}/content`
 				);
 
-				await writeFile(
-					path.join(creationDirectory, "./src/index.js"),
-					dashScript
-				);
+				for (const file of dashScript) {
+					await writeFile(
```

Comment:
Doing this in parallel (i.e. with await Promise.all()) might be a nice optimisation